core-chart/templates/postcreatehooks/wds.yaml

```diff
@@ -215,6 +215,10 @@ spec:
             - "{{"transport_cp_name={{.ITSName}}"}}"
             - --set
             - "verbosity={{.Values.verbosity.transport | default .Values.verbosity.default | default 4 }}"
+            - --set
+            - "max_num_wrapped={{.Values.transport_controller.max_num_wrapped}}"
+            - --set
+            - "max_size_wrapped={{.Values.transport_controller.max_size_wrapped}}"

```

Comment:
I do not see these two names in the values of the ocm-transport-plugin chart.

```shell
mspreitz@mjs13 kubestellar % helm show values oci://ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin --version 0.1.11
Pulled: ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin:0.1.11
Digest: sha256:b033717574693035b867e958641a7132d5c1baeb6599e29d1410badb926ec82b
# Copyright 2024 The KubeStellar Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

# Set the name of the transport control plane
transport_cp_name: ""

# Set the name of the WDS control plane
wds_cp_name: wds1

# Set the controller verbosity
verbosity: 4

# [host]:port at which to listen for HTTP GET /metrics
metrics_bind_addr: ":8090"

# [host]:port at which to listen for HTTP GET /debug/pprof
pprof_bind_addr: ":8092"

# QPS and burst for accessing the ITS
transport_qps: 5
transport_burst: 10

# QPS and burst for accessing the WDS
wds_qps: 5
wds_burst: 10
```

FYI: #2433 moves that source and publisher of that chart from ks/ocm-transport-plugin to ks/ks.